### ISSUE TITLE
[GH Request] Repo for https://github.com/open-craft/xblock-skill-tagging

### ISSUE THREAD

**Issue by irfanuddinahmad** [openedx/tcril-engineering#687]
### Firm Name

2U

### Urgency

Medium (< 2 weeks)

### Requested Change

New repo for https://github.com/open-craft/xblock-skill-tagging

### Reasoning

Need to push releases to PyPI

**Comment by bmtcril** [comment]
Hi @irfanuddinahmad just to confirm, this is a request for tCRIL to transfer the repo into the openedx org? Is this code that OpenCraft owns, or has is it owned by 2U under a development agreement? I only ask because the firm name on the ticket is "2U".

**Comment by irfanuddinahmad** [comment]
Hi @bmtcril 
This work is the outcome of a blended project between 2U/edX and OpenCraft.

**Comment by bmtcril** [comment]
@irfanuddinahmad to transfer ownership of this repo we will need to follow the directions listed here: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3438903337/On-call+Playbooks#%F0%9F%8F%97-Transferring-repositories-into-openedx . Would you all be comfortable giving me admin access so we can transfer instead of forking this? 

We will need an affirmative statement from a manager or principal engineer from whichever organization actually owns the code that's being transferred along the lines of: 

"I confirm that <org> is intentionally submitting the repo code as a contribution to the Open edX public platform codebase under the existing <org>-tCRIL license and contribution agreements."

@antoviaque usually handles this on the OpenCraft side.

**Comment by antoviaque** [comment]
@bmtcril Thanks for the ping - yes, I can confirm that OpenCraft is "intentionally submitting the repo code as a contribution to the Open edX public platform codebase under the existing - tCRIL license and contribution agreements." CC @navinkarkera

**Comment by bmtcril** [comment]
Great, thanks. So I think the next step is to grant me admin on the repo and I'll do the transfer.

**Comment by antoviaque** [comment]
@bmtcril Done! Don't hesitate if you need anything else.

**Comment by bmtcril** [comment]
Thanks all, I've done the transfer and will start setting up permissions. If there is anyone who would like / need core contributor access to the repo, feel free to open a thread on the forums and I'll send my approval.

**Comment by irfanuddinahmad** [comment]
@bmtcril   we want to merge https://github.com/openedx/xblock-skill-tagging/pull/9 ... however, we don't seem to have the rights to do so

**Comment by bmtcril** [comment]
@irfanuddinahmad @sameenfatima78 sorry I missed the original question. Are there existing core contributors who we can expand access to this repo for or are you looking to onboard new core contributors with access here?

**Comment by irfanuddinahmad** [comment]
> @irfanuddinahmad @sameenfatima78 sorry I missed the original question. Are there existing core contributors who we can expand access to this repo for or are you looking to onboard new core contributors with access here?

@bmtcril Thanks. Can you add me and Sameen.

**Comment by bmtcril** [comment]
@irfanuddinahmad @sameenfatima78 so neither of you are actually in the core contributor program, my understanding is that you are working on this project in a 2U capacity under their CLA? I think @antoviaque as a representative of the owning organization would need to designate you as maintainers here since this isn't a historically inherited 2U repo.

Presuming you want to be the maintainers of this project and all that entails. :)

**Comment by irfanuddinahmad** [comment]
@bmtcril you are correct, we are working under the 2U CLA.

**Comment by antoviaque** [comment]
@bmtcril Happy to help with this as needed, for the nomination process, or anything related to it.

@irfanuddinahmad @sameenfatima78 Are you looking to become maintainers as per https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0055-proc-project-maintainers.html#abstract ? If so, I could guide you through the [process to become maintainer](https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0055-proc-project-maintainers.html#becoming-a-maintainer), if you want?

**Comment by irfanuddinahmad** [comment]
@antoviaque At the moment we just want to release the initial version to PyPI so that we get unblocked on our work. It would be an honor to maintain this codebase, however, we donot currently have the bandwidth to fulfill all of the maintainer responsibilities.

**Comment by bmtcril** [comment]
Ok, I don't want to keep you all blocked on this so I can push out a release, but I can't become the maintainer of this repo simply because I pushed the button to bring it into the organization. I won't have the context to make appropriate decisions about it or review changes.

The version is currently at 0.1.0, is the code on main now what you want pushed out or do you want that other PR merged first?

**Comment by irfanuddinahmad** [comment]
Thanks @bmtcril 
Yes. main can be released.

**Comment by bmtcril** [comment]
@irfanuddinahmad it looks like it already was, back in March: https://pypi.org/project/skill-tagging/ the only changes since then have been to Github actions. I think if there's anything more to be done here you should open a new ticket since we're well past the original request.
